docs/usage/executor/kubectl.md

```diff
@@ -4,22 +4,18 @@ title: "Kubectl"
 sidebar_position: 2
```

Comment:
Please also go through other docs to ensure we don't have old syntax. For example, I found:
<img width="859" alt="image" src="https://user-images.githubusercontent.com/7155799/210558931-c6f958cf-0fba-43cb-ba76-9097034061fa.png">

Reply:
fixed 👍